core/src/main/scala/janstenpickle/scala/syntax/syntax.scala

```diff
@@ -24,95 +25,122 @@ object catsconversion {
 
   implicit def validatedApplicative: Applicative[Shit] =
     new Applicative[Shit] {
-      def ap[A, B](f: Shit[A => B])(fa: Shit[A]): Shit[B] =
+      def ap[A, B](f: Shit[A ⇒ B])(fa: Shit[A]): Shit[B] =
         (fa, f) match {
-          case (Ok(a), Ok(fab)) => Ok(fab(a))
-          case (Fail(e1), Fail(e2)) => Fail(e1 ++ e2)
-          case (Ok(_), i@Fail(_)) => i
-          case (i@Fail(_), Ok(_)) => i
+          case (Ok(a), Ok(fab)) ⇒ Ok(fab(a))
+          case (Fail(e1), Fail(e2)) ⇒ Fail(e1 ++ e2)
+          case (Ok(_), i@Fail(_)) ⇒ i
+          case (i@Fail(_), Ok(_)) ⇒ i
         }
 
       def pure[A](x: A): Shit[A] = Result.ok(x)
 
-      override def product[A, B](fa: Shit[A],fb: Shit[B]): Shit[(A, B)] = ap(map(fa)(a => (b: B) => (a, b)))(fb)
+      override def product[A, B](
+        fa: Shit[A],
+        fb: Shit[B]
+      ): Shit[(A, B)] = ap(map(fa)(a ⇒ (b: B) ⇒ (a, b)))(fb)
 
-      override def map[A, B](fa: Shit[A])(f: A => B): Shit[B] = ap(pure(f))(fa)
+      override def map[A, B](fa: Shit[A])(f: A ⇒ B): Shit[B] = ap(pure(f))(fa)
     }
 
-  val shit = (Result.fail[List[String], Int](List("sdfsf")) |@| Result.fail[List[String], Int](List("3423rwfef")) |@| Result.ok[List[String], String]("sdfsf")).map((x, y, z) => (x, y, z))
+  val shit = (
+    Result.fail[List[String], Int](List("sdfsf")
+  )
+  |@| Result.fail[List[String], Int](List("3423rwfef"))
+  |@| Result.ok[List[String], String]("sdfsf")).map((x, y, z) ⇒ (x, y, z))
 }
 
-object option {
+object CatsOption {
```

Comment:
The reason for calling the object above here `CatsConversion` was for converting from cats types to uscala, here the code is just providing syntax for `Option`, so can just be called `Option` rather than `CatsOption`.